modules/standard/Errors.chpl

```diff
@@ -597,7 +597,7 @@ module Errors {
   pragma "always propagate line file info"
   proc assert(test: bool) {
     if !test then
-      __primitive("chpl_error", c"assert failed");
+      __primitive("chpl_error", "assert failed".c_str());
```

Comment:
hmm. What happens if you just have a regular string literal here and not `c"bla bla"` and not `"bla bla".c_str()`

Reply:
I would need some help debugging this compilation error:
```
Operand is null
  call void @chpl_error(<null operand!>, i32 noundef %16, i32 noundef %15) #0
internal error: LLVM function verification failed [codegen/cg-symbol.cpp:2825]
```

Reply:
I was able to remove the `"StringLiteral".c_str()` pattern most everywhere but here, when it's used in a call to `__primitive()`.

Reply:
OK, using `.c_str()` here sounds OK to me in that case. It should be possible to adjust the primitive implementation, but if it's specific to a literal used in a primitive, adding `.c_str()` seems fine for now.